Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Applications/CalculatorTest.cs

```diff
@@ -7,25 +7,27 @@ namespace Aquality.WinAppDriver.Tests.Applications
 {
     public class CalculatorTest : TestWithApplication
     {
+        private readonly CalculatorWindow CalculatorWindow = new CalculatorWindow();
```

Comment:
please start name of the private non-static property with the lowercase letter

Reply:
done